src/components/Checkbox/Checkbox.tsx

```diff
@@ -10,15 +10,24 @@ interface ICheckbox {
   onChange?: (value: any) => void;
```

Comment:
The same here, onChange should not be optional. Do we use it as optional in our whitelabel?